File: 1. 23.08/python/11.py
space = 2
c = 1
for i in range(1,8):
    for i in range (0,space):
        print(' ',end="")
    for i in range(0, c):
        print('*',end="")
    print('\n')
    if space > 0:
        space -= 1
    if c < 5:
        c += 2;

File: 1. 23.08/C++/11.cpp
#include <iostream>

int main()
{
	for(int i=1, space=2, c=1; i < 8; ++i)
	{
		for(int i = 0; i < space; ++i)
			std::cout<<' ';
		for(int i = 0; i < c; ++i)
			std::cout<<'*';
		std::cout<<std::endl;
		if (space > 0)
			--space;
		if (c < 5)
			c += 2;
	}
    return 0;
}
